haskell-api.md

```diff
@@ -117,6 +117,97 @@ possible. Issues include:
 If you are creating a new `PUT` consider if it could be expressed in a more
 granular `PATCH` semantic.
 
+### `PATCH` semantics
+
+Modifications to an existing resource should be made via a `PATCH` request.
+
+### Specific guidelines
+
+A well-formed `PATCH` endpoint
+
+- Must update an entity's `updatedAt` field, if present.
+- Should return the modified resource. The resource should be re-fetched for
```

Comment:
"re-fetched" is a bit confusing to me as I assume you're talking about refreshing from the database and not refetching from a different endpoint. Could just be me, but we have `refreshEntity` and `refreshRecord` in the Persistent module so maybe this should use similar verbiage

Reply:
Clarified in recent commit, thanks!